### PR TITLE
Remove "work in progress" from the description

### PR DESCRIPTION
**Pull request by gabro**
Followup of https://github.com/scalameta/metals/pull/782

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "metals",
   "displayName": "Scala (Metals)",
-  "description": "Work in progress language server for Scala",
+  "description": "Language Server for Scala",
```
Comment (olafurpg) [inline package.json]: I feel like we should come up with a slightly more detailed description first. Something like “Language server for Scala with rich IDE features”. I’m totally sold on removing the WIP title and updating it everywhere (website, marketplace,...)

Comment (gabro) [reply package.json]: Isn't it a bit redundant? Other language servers follow pretty much the same pattern: "X language server", e.g. https://github.com/eclipse/eclipse.jdt.ls

Comment (olafurpg) [reply package.json]: It would be nice to have “IDE” in the title to help match search queries

Comment (tgodzik) [reply package.json]: We could add that to keywords

Comment (gabro) [reply package.json]: With that in mind I'm 👍 for "Scala language server with rich IDE feature" and also adding "ide" to the keywords

**Comment by gabro** [comment]
Better?

**Pull request merged**